File: classical/PRIME1.cpp
#include <iostream>


bool isPrime(int number) {
    if (number < 2) {
        return false;
    }
    else if (number == 2) {
        return true;
    }
    else {
        for (int i = 2; i * i <= number; i++) {
            if (number % i == 0) {
                return false;
            }
        }
    }

    return true;
}


int main() {
    int loops;
    int start, stop;

    std::cin >> loops;

    for (int i = 0; i < loops; i++) {
        std::cin >> start >> stop;

        for (int j = start; j <= stop; j++)
            if (isPrime(j))
                std::cout << j << std::endl;

        std::cout << std::endl;
    }

    return 0;
}

File: PRIME1.py
def is_prime(number: int):
    if number < 2:
        return False
    else:
        i = 2
        while i*i <= number:
            if number % i == 0:
                return False
            i += 1
    return True


if __name__ == "__main__":
    loops = int(input())
    for _ in range(loops):
        first, last = (int(i) for i in input().split())
        for i in range(first, last + 1):
            if is_prime(i):
                print(i)
        print()

# Time limit exceeded!